tests/testthat/test-variable-order.R

```diff
@@ -830,4 +842,76 @@ with_test_authentication({
         expect_identical(urls(ordering(ds)[["Group 2.5"]]),
             urls(variables(ds[c("v4", "v6")])))
     })
+
+    test_that("copyOrder copies across datasets with simple order", {
```

Comment:
I've added a test on an alternate target dataset where the target is missing some variables in the target and the target has some extra variables that the source does not have. Additionally, the ids for the common variables are different, so we're checking that the ids aren't what are being used.